src/app.ts

```diff
@@ -4,6 +4,7 @@ import { NOT_FOUND } from 'http-status';
 import { appConfiguration, AppDataSource } from './config';
 import { loggerConfiguration } from './utils';
 import bodyParser from 'body-parser';
+import { router } from './routes/router';
```

Comment:
move all src to /v1 folder and manage v1 there? like they did in obsrv - as discussed yesterday

Reply:
mentioned changes resolved.